### DISCUSSION TITLE
Does the new version has `Not now, show schematics list` option?

### DISCUSSION THREAD

**Discussion by doggy8088** [cyrilletuzi/vscode-angular-schematics#912] [Q&A] [answered]
The Angular Schematics v5.4.2 contains a option to list all the existing schematics list without the need to have Pro edition.

![image](https://github.com/cyrilletuzi/vscode-angular-schematics/assets/88981/009e9fa9-ca27-4b99-a808-00d5c42ed629)

In the v6.0.5, the option is gone. Does that mean anyone who installed your extension must be purchase a Pro license to use?

**Comment by cyrilletuzi** [comment, marked as answer]
Hello, no, this view has not been locked under the Pro edition. It is just that the extension has been nearly completely rewritten, and I did not find worth it to add this view back.

This schematics list was a vestige from the very first versions of the extension, and served only to make the extension discoverable (as some people may not the see right away the "Generate" command when right-clicking in the Explorer). But now that this panel has another content, and that a full walkthrough / documentation / tutorial  is available in the extension, the schematics list did not seem useful anymore.

Indeed, like it was stated at the top of this schematics list, launching the generation command from a right-click in the Explorer is way more simpler and quicker than from this schematics list, as in the latter case, you have to type the whole path where you want to generate your component (or else). It is exactly one of the main goals of this extension, compared to the Angular CLI, to not have to do that.

What was your use case?

**Reply by doggy8088** [reply]
I just can't use the tool. It always show up a page wants me to buy Pro edition. See below:

https://github.com/cyrilletuzi/vscode-angular-schematics/assets/88981/bbd68b33-62a6-452c-b7db-872038ae3c5a

It's not obvious to use the extension to create a new new component now. I don't even know I have to choose `CommonModule` at first time.

![image](https://github.com/cyrilletuzi/vscode-angular-schematics/assets/88981/94855435-0517-4fb3-95ff-b8f0909418c2)

**Comment by cyrilletuzi** [comment]
> I just can't use the tool

That is not true. Just click on "CommonModule", or just press "Enter", and it works: 1. perfectly fine 2. exactly as before and as in the Angular CLI, where the CommonModule is also the default.

> It always show up a page wants me to buy Pro edition

That is not true either. The page shows up because you click on a Pro edition feature, which is very clearly represented by the diamond icon. It is explained everywhere (on the page that is showed up, in the walkthrough displayed on first install of the extension, on the panel view you talked about in your first message, in the documentation both in the extension and on GitHub....) what it means.

Nothing forces you to buy the Pro edition if you do not want to support my work. Marking this as answered and closing it, as it is starting to cross the line of non constructive criticism.
